tests/lac/tensor_product_matrix_05.cc

```diff
@@ -0,0 +1,117 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// Similar to tensor_product_matrix_02.cc unless testing with
```

Comment:
Do you want to say *except* here rather than *unless*?

Reply:
This pattern is also used in the other tests' descriptions.

Reply:
Yes, it should be "except".